src/Dialog/Dialog.test.tsx

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright (c) 2015 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import React from 'react';
+import { fireEvent, screen } from '@testing-library/react';
+
+import render from '../../test/testrenderer';
+import {
+    ConfirmationDialog,
+    Dialog,
+    DialogButton,
+    ErrorDialog,
+    InfoDialog,
+} from './Dialog';
+
+const noop = jest.fn();
+describe('Dialog', () => {
+    const dialog = (isVisible = true) => (
+        <Dialog isVisible={isVisible}>
+            <Dialog.Header title="Test Title" />
+            <Dialog.Body>Test Text</Dialog.Body>
+            <Dialog.Footer>
+                <DialogButton onClick={noop}>Close</DialogButton>
+            </Dialog.Footer>
+        </Dialog>
+    );
+
+    describe('is visible when it', () => {
+        it('is rendered when visible', () => {
```

Comment:
'Dialog is visible when it is rendered when visible' ?

Reply:
Fixed